src/utills/checkCookie.js

```diff
@@ -0,0 +1,24 @@
+const checkCookie = (name) => {
+  if (!name) {
+    return;
+  }
+
+  const cookies = document.cookie ? document.cookie.split("; ") : [];
+  const cookieStorage = {};
+
+  for (let i = 0; i < cookies.length; i++) {
+    const parts = cookies[i].split("=");
+    const value = parts.slice(1).join("=");
+    const found = decodeURIComponent(parts[0]);
+
+    cookieStorage[found] = value;
+
+    if (name === found) {
+      break;
+    }
+  }
+
+  return cookieStorage[name] ? true : false;
```

Comment:
boolean 값을 반환하는 삼항연산자의 사용은 지양하는 것이 좋습니다.

```suggestion
  return !!cookieStorage[name];
```